<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCUWidget</class>
 <widget class="QWidget" name="DCUWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background:black;
}
/***页码****/
QLabel#label_pageNum{
	color:white;
	font:16px;
	border:1px solid white;
	border-top-left-radius:4px;
	border-bottom-left-radius:4px;
	border-right:none;
}
/*****总页数*****/
QLabel#label_totalPage{
	color:white;
	font:16px;
	border:1px solid white;
	border-bottom-right-radius:4px;
	border-top-right-radius:4px;
	border-left:none;
}

QPushButton{
	color:#f9f900;
	font:75 16px;
	border:1px solid  #fafa00;
	border-radius:4px;
}

QPushButton:focus:!active{
	background:transparent;
}

QPushButton:pressed:active,QPushButton:focus:active{
	background:#0000fe;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>470</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_pageNum">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>33</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>33</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_totalPage">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>33</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>33</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>55</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QPushButton" name="pbt_pageUp">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>33</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>33</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>上一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbt_pageDown">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>33</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>33</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>下一页</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>stackedWidget</zorder>
  <zorder>horizontalSpacer_4</zorder>
  <zorder>pbt_pageUp</zorder>
  <zorder>pbt_pageDown</zorder>
  <zorder>horizontalSpacer_4</zorder>
  <zorder>pbt_pageUp</zorder>
  <zorder>pbt_pageDown</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
